share/spice/just_delete_me/footer.handlebars

```diff
@@ -0,0 +1,10 @@
+{{#if delete_url}}
+    <div class="one-line">
+         <a href='{{delete_url}}'>{{delete_url}}</a>
```

Comment:
I agree on the small tiles it makes sense to just have the descriptive text as you say, though the link might not be directly to the deletion page, as can be seen on IRCCloud on your screenshot.